gateway/store/store.go

```diff
@@ -782,6 +794,17 @@ func (s *k8sStore) ListVirtualService() (l7vs []*v1.VirtualService, l4vs []*v1.V
 							location = &v1.Location{
 								Path:          path.Path,
 								NameCondition: map[string]*v1.Condition{},
+								Rewrite:       rewrite.Config{},
+							}
+							i, err := rewrite.NewParser(s).Parse(&ing.ObjectMeta)
+							if err == nil {
+								logrus.Warningf("Uxpected error with ingress: %v", err)
+								cfg := i.(*rewrite.Config)
```

Comment:
Refer to the comments at line 146, Check similar implementations for modifications